server.js

```diff
@@ -24,36 +25,61 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, "./welcome-screen")));
 
 app.use((req, res, next) => {
-    res.append("Access-Control-Allow-Origin", [ "*" ]);
-    res.append("Access-Control-Allow-Methods", "GET,PUT,POST,DELETE");
-    res.append("Access-Control-Allow-Headers", "Content-Type");
-    next();
+  res.append("Access-Control-Allow-Origin", ["*"]);
+  res.append("Access-Control-Allow-Methods", "GET,PUT,POST,DELETE");
+  res.append("Access-Control-Allow-Headers", "Content-Type");
+  next();
 });
 
 const checkCredentials = async () => {
-    try {
-        await client.vouchers.list();
-    } catch (error) {
-        if (error.code === 401) {
-            const msg = "Your API credentials are incorrect, please check your applicationId and secretKey or visit `https://docs.voucherify.io/docs/authentication` to complete your app configuration.";
-            throw new Error(msg);
-        }
-        throw new Error(error);
+  try {
+    await client.vouchers.list();
+  } catch (error) {
+    if (error.code === 401) {
+      const msg = "Your API credentials are incorrect, please check your applicationId and secretKey or visit `https://docs.voucherify.io/docs/authentication` to complete your app configuration.";
+      throw new Error(msg);
     }
+    throw new Error(error);
+  }
 };
+
+/*
+Important!
+The 'Reward Promotion' campaign is required for this project to work properly.
+If it doesn't exist you will see an error in the console.
+Visit https://github.com/voucherifyio/voucherify-examples/tree/main/tiered-cart-promotions#creating-a-reward-promotion-campaign 
+for getting instructions on how to create the missing campaign.
+*/
+
+const checkCampaign = async () => {
```

Comment:
The `checkCampaign` function actually adds a campaign to the V% account if it does not exists, which may be a little dangerous if someone by accident connects the production account. I suggest does not allow to run the example (like for `checkCredentials`) and displaying the message that you have already in the catch block.